wiki/templates/wiki/base_site.html

```diff
@@ -66,6 +66,11 @@
                     </a>
                   </li>
                   {% include "wiki/plugins/notifications/menubaritem.html" %}
+                  {% with tpl_name="wiki/plugins/globalhistory/menubaritem.html" %}
```

Comment:
This is an interesting way of discovering the plugin. We could also create some other sort of template tag, like a `wiki_plugin_enabled` that takes a plugin name as a string and tests if its in INSTALLED_APPS.

Because in case we do a template refactor, the above will break, which might not be very obvious in the future..